skfem/models/general.py

```diff
@@ -11,9 +11,9 @@ def divergence(u, v, w):
 
 @LinearForm
 def rot(v, w):
-    return dot(curl(v), w.w)
+    return dot(curl(v), w["w"])
```

Comment:
Now that we have the option to use pretty much any name in place of `"w"` (I belive when this was originally implemented, it was only `"w"` allowed), the choice of `"w"` feels rather arbitrary. I wonder if we could improve these forms somehow by keeping that in mind?

Reply:
See #550

Reply:
I see, yes, that's a good idea, a less arbitrary name than `"w"`.  But then how is the user to know or remember it?  Perhaps better still is indeed the [tacit](https://en.wikipedia.org/wiki/Tacit_programming) style enabled by #550. Does that mean deprecating `rot` and `vrot`?  I suppose that if they're left in `skfem.models.general` at least temporarily during deprecation then the attribute &rarr; item fix is still required.

I think `rot` needs a different `BilinearForm` to `vrot`.